File: app/src/main/java/dev/cyberarm/renegade_server_list/library/ServerSettings.java
package dev.cyberarm.renegade_server_list.library;

import java.util.ArrayList;

public class ServerSettings {
    public String ID;
    public int notifyPlayerCount;
    public ArrayList<String> notifyMapNames, notifyUsernames;

    public ServerSettings(String ID, int notifyPlayerCount, ArrayList<String> notifyMapNames, ArrayList<String> notifyUsernames) {
        this.ID = ID;
        this.notifyPlayerCount = notifyPlayerCount;
        this.notifyMapNames = notifyMapNames;
        this.notifyUsernames = notifyUsernames;
    }
}

File: app/src/main/java/dev/cyberarm/renegade_server_list/settings/data/ServerSettings.kt
package dev.cyberarm.renegade_server_list.settings.data

data class ServerSettings(
    val schema: Int,
    val uuid: String,
    val name: String,
    val notifyPlayerCount: Int,
    val notifyMapNames: ArrayList<String>,
    val notifyUsernames: ArrayList<String>,
    val notifyRequireMultipleConditions: Boolean
)
